test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14002,6 +13989,56 @@ def add_mom_dyn_res(self, custom_resource, script_body=None,
         self.add_config(a)
         return res_file
 
+    def enable_cgroup_cset(self):
+        """
+        Configure and enable cgroups hook
+        """
+        # check if cgroups subsystems including cpusets are mounted
+        with open(os.path.join(os.sep, 'proc', 'mounts'), 'r') as fd:
```

Comment:
If we're talking about a remote mom, you currently read the file on the local host. You need to use get the mounts from the remote machine.

Reply:
I switched to using json. We are on a local host.

Reply:
If you have two moms, one of them is probably on a remote host. However, you'll still be checking the local host for the cgroups mounts.

Reply:
Not only that, if I specify a mom list that doesn't include the local host, it will only work if the systems are identical.